### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by WDHudson**
The majority of the review is for index.jsx. All of the other changes are either changes in versions of libraries in package.json and cosmetic changes for CSS. The changes worth noting are:

- I stylized some of the components in the Items.jsx file. 
- in Index.jsx, the rotateMedia function is what rotates the white border around the thumbnail picture when one is clicked and renders that thumbnail to the larger viewer above. It moves to the next picture every 4 seconds. 

All of the other changes are CSS related and the scroll bar below is a WIP. It's really just that rotateMedia function that needs to be reviewed.

### CODE REVIEW

File: client/index.jsx [hunk 1]
```diff
@@ -1,37 +1,67 @@
...
+  rotateMedia(pic) {
+    var array = this.state.mediaRoll
+    this.setState({ largePlayer: array[pic].url, activeItem: pic })
+
+    clearInterval(loopPictures)
+
+    var loopPictures = setInterval(() => {
+      if (this.state.largePlayer === array[array.length - 1].url) {
+        this.setState({ largePlayer: array[2].url, activeItem: 1 })
+      } else {
```
Comment (damienmjones) [inline client/index.jsx:47]: Is this array zero-based or one-based?

File: client/index.jsx [hunk 2]
```diff
@@ -1,37 +1,67 @@
...
+        this.setState({ largePlayer: array[this.state.activeItem + 1].url })
+      }
+      this.setState({ activeItem: this.state.activeItem + 1 })
+    }, 4000)
+
```
Comment (damienmjones) [inline client/index.jsx:52]: I think the problem here is that loopPictures doesn't persist between clicks.

File: client/Items.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  position: relative;
+  background-size: contain;
+  float: left;
+
+  iframe, img {
+    height: 65px;
+    width: 115px;
+    border: none;
+    background-image: linear-gradient(135deg,  #3d6c8d 0%,#2e5470 100%);
+  }
```
[the remark this answers is not in the document]
Comment (WDHudson) [reply client/Items.jsx:14]: I did, but the iframe is just for the videos, not for the images. The differences created a number of different issues.

File: client/index.jsx (hunk 1 above)
Comment (WDHudson) [reply client/index.jsx:47]: Zero based. When the pictures start rotating, it only rotates to the pictures, when it reaches the end of the thumbnail array, it goes to the first picture, not the first video.

File: client/index.jsx (hunk 2 above)
Comment (WDHudson) [reply client/index.jsx:52]: Ahhh good catch! I'll change that.

**Pull request merged**